src/Stripe.net/Entities/Sources/SourceSourceOrderItem.cs

```diff
@@ -0,0 +1,39 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class SourceSourceOrderItem : StripeEntity<SourceSourceOrderItem>
+    {
+        /// <summary>
+        /// The amount (price) for this order item.
+        /// </summary>
+        [JsonProperty("amount")]
+        public long Amount { get; set; }
```

Comment:
This attribute is flagged as nullable in the spec, so should be `long?`.

Reply:
Updated to nullable.